migrations/tenant/0019-alter-default-value-objects-id.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE storage.objects ALTER COLUMN id SET DEFAULT gen_random_uuid();
```

Comment:
should be safe to run on large tables?

Reply:
Changing the default value of a column in PostgreSQL typically does not require a lock that would prevent concurrent accesses to the table. When you alter a column to set a default value, PostgreSQL does not need to rewrite the table or the existing rows; instead, it just changes the metadata of the table to specify the new default for future inserts.

However, it's important to note a few things:

- Brief Access Exclusive Lock: The ALTER TABLE command acquires a brief ACCESS EXCLUSIVE lock on the specific table, which blocks all other accesses (reads and writes) to the table until the command is complete. But because changing the default value is usually a quick operation, the lock is typically very brief and may not be noticeable.

- No Impact on Existing Rows: Changing the default value of a column does not affect existing rows in the table. The new default value is only used for new rows that are inserted into the table without a specific value for that column.